src/text-tags.js

```diff
@@ -1,7 +1,7 @@
 import {get, isEmpty} from 'lodash';
 import {objectToTags} from './utils';
 
-function buildTagsFromStory(config, story, url = {}) {
+function buildTagsFromStory(config, story, url = {}, data = {}) {
```

Comment:
better to rename the param, so that it can be more explicit

Reply:
the data here represent all the data that is coming by the response, it does not only have SEO data but it has config, API response, etc. so I'm calling it as data.